### ISSUE TITLE
Unable to Upload Video via Frontend in Docker - "Uh oh, we cannot process this video" Error

### ISSUE THREAD

**Issue by NgdHung31** [facebookresearch/sam2#361]
First off, thank you for creating this awesome project! I'm really enjoying using it.

I've encountered an issue when trying to upload my own video through the frontend. While I can successfully use my video by placing it in the `demo/data/gallery` folder and starting the container, I run into a problem when trying to upload it directly.

Specifically, after selecting "Upload" in the frontend, I receive the following error message: 

> Uh oh, we cannot process this video. Please upload another video, and make sure that the video’s file size is less than 70Mb.

I've double-checked the file size, and it is definitely under the 70MB limit. I'm wondering if this is a known issue or if anyone else has experienced something similar.

Any help or guidance would be greatly appreciated!

**Comment by NgdHung31** [comment]
Hi again,
Thanks for the quick reply!

I've done some further testing and can confirm that this happens with **all videos** I try to upload, even those around 1MB in size.
The terminal output when I attempt to upload a video is consistently:

```
backend-1  |  <my ip and my date, time> "POST /graphql HTTP/1.1" 400 24 "http://localhost:7262/" "Mozilla/5.0 (X11; Linux x86_64)
```

As mentioned before, I've tried using the provided sample videos and they work fine, so I believe **ffmpeg is functioning correctly**.

Interestingly, after looking through the source code, I don't think the issue lies within the **3 file processing conditions** in `useUploadVideo.ts` (format, size, and count checks).

Based on my observations, it seems like the problem might be in the **data flow between the frontend and backend**.  Perhaps there's an error in how the video data is being sent, or the backend is unable to receive/process the uploaded video data correctly.

I can provide more information about my Docker environment if needed. Any further guidance would be greatly appreciated!

Thanks!

**Comment by raedle** [comment]
We took a look on a fresh conda env install, and the issue is reproducible. The underlying issue was a breaking change introduced by a dependency. A forward fix is submitted in PR #364

**Comment by NgdHung31** [comment]
That's fantastic, thank you so much for the quick investigation and fix! I'll check out PR #364 and test it out in my environment. I really appreciate your responsiveness and dedication to this project.
